hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1270,7 +1270,14 @@
       create pipelines.
     </description>
   </property>
-
+  <property>
+    <name>ozone.scm.auto.create.factor.one.pipeline</name>
```

Comment:
I think `ozone.scm.pipeline.creation.auto.factor.one` would better match hierarchical config naming pattern.  Which also helps keep related configs together when refactoring to `@Config` annotation-based implementation, since `@ConfigGroup` requires a common prefix.

Reply:
Agree, another proposal for the key name to be ozone.scm.auto.create.factor.one.pipeline, this might not follow the same common prefix but better readability.

Reply:
Sounds good. Updated.

Reply:
Thanks